.github/workflows/pull-compliance.yml

```diff
@@ -0,0 +1,145 @@
+name: "Pull: Compliance Tests"
+
+on: [pull_request]
+
+jobs:
+  lint_basic:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.20'
+      - name: deps-backend
+        run: make deps-backend deps-tools
+      - name: lint backend
+        run: make lint-backend
+        env:
+          GOPROXY: https://goproxy.io # proxy.golang.org is blocked in China, this proxy is not
+          GOSUMDB: sum.golang.org
+          TAGS: bindata sqlite sqlite_unlock_notify
+  lint_windows:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.20'
+      - name: deps-backend
+        run: make deps-backend deps-tools
+      - name: lint-backend-windows
+        run: make lint-go-windows lint-go-vet
+        env:
+          GOPROXY: https://goproxy.io # proxy.golang.org is blocked in China, this proxy is not
+          GOSUMDB: sum.golang.org
+          TAGS: bindata sqlite sqlite_unlock_notify
+          GOOS: windows
+          GOARCH: amd64
+  lint_gogit:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: setup go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '>=1.20'
+      - name: deps-backend
+        run: make deps-backend deps-tools
+      - name: lint-backend-gogit
+        run: make lint-backend
+        env:
+          GOPROXY: https://goproxy.io # proxy.golang.org is blocked in China, this proxy is not
+          GOSUMDB: sum.golang.org
+          TAGS: bindata gogit sqlite sqlite_unlock_notify
+      - name: checks backend
+        run: make --always-make checks-backend # ensure the 'go-licenses' make target runs
```

Comment:
`--always-make` already means "ensure the target run" 😁   so the comment could be removed?